Cartfile

```diff
@@ -1,2 +1,2 @@
-github "BranchMetrics/ios-branch-deep-linking" ~> 0.31.2
+github "BranchMetrics/ios-branch-deep-linking" ~> 0.34.0
```

Comment:
let's open this up to the latest

Reply:
0.34 is the latest. I think we discussed this but I want to clarify